### PR TITLE
moved code into fund.go from fineract_client.go

### PR DESCRIPTION
**Pull request by sapanakothari**

### CODE REVIEW

File: fund.go [hunk 1]
```diff
@@ -10,6 +10,13 @@ import (
 )
 
 const (
+	fineractHost     = "https://13.209.34.65:8443" //"https://demo.openmf.org"
```
Comment (poorva29) [inline fund.go]: This could have been left out in common, all the sub modules need this, please move this to `fineract_client.go`

File: fund.go [hunk 2]
```diff
@@ -258,3 +265,25 @@ func fundsURLParams(officeId string) string {
...
+
+func savingsAccountsURL() string {
+	return path.Join(baseURL, "savingsaccounts")
+}
+
+func headOfficeURL() string {
+	return path.Join(baseURL, "offices")
+}
+
+func NewMockClient() (*Client, error) {
```
Comment (poorva29) [inline fund.go]: Same for this too, please move it to `fineract_client.go`

Comment (sapanakothari) [reply fund.go]: ok

Comment (sapanakothari) [reply fund.go]: incorporated.

**Pull request merged**